roles/nuage_master/tasks/serviceaccount.yml

```diff
@@ -1,32 +1,19 @@
 ---
-- name: Create temporary directory for admin kubeconfig
-  command: mktemp -u /tmp/openshift-ansible-XXXXXXX.kubeconfig
-  register: nuage_tmp_conf_mktemp
-  changed_when: False
-
-- set_fact:
-    nuage_tmp_conf: "{{ nuage_tmp_conf_mktemp.stdout }}"
-
-- name: Copy Configuration to temporary conf
-  command: >
-    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{nuage_tmp_conf}}
-  changed_when: false
+- name: Load lib_openshift modules
```

Comment:
We probably need to get consensus on this approach sooner rather than later, since it seems like the include_role/role dependency discussions are happening more frequently.

I would suggest this be moved to role dependencies here and in the openshift_manageiq